hal/src/stm32f2xx/core_hal_stm32f2xx.c

```diff
@@ -314,6 +319,11 @@ void HAL_Core_Config(void)
 #if PLATFORM_ID==8 // Additional pins for P1
     for (pin_t pin=24; pin<=29; pin++)
         HAL_Pin_Mode(pin, INPUT);
+    const uint8_t* data = (const uint8_t*)dct_read_app_data(DCT_RADIO_FLAGS_OFFSET);
+    uint8_t current = (*data);
+    if ((current&3) == 0x2) {
```

Comment:
I know this is my (shoddy!) code, and felt the same way when I wrote it....it's probably best to pull these magic numbers into a function like

```
bool isWiFiPowersaveClockDisabled() {
 const uint8_t* data = (const uint8_t*)dct_read_app_data(DCT_RADIO_FLAGS_OFFSET);
  uint8_t current = (*data);
  return ((current&3) == 0x2);
}
```